tests/README.md

```diff
@@ -44,6 +44,29 @@ tests
 
 ## Contibute to terraform rhcs provider tests
 Please read the structure and contribte code to the correct place
+### Contribute to day1 case
+* Define manifests files under terraform-provider-rhcs/tests/tf-manifests/rhcs/clusters
+* Define profile in terraform-provider-rhcs/tests/ci/profiles/tf_cluster_profile.yml
+* If any other data can be created by terrafom provder, define steps under terraform-provider-rhcs/tests/tf-manifests/<provider>/<resource name>
+* Define prepare steps according to the profile in terraform-provider-rhcs/tests/ci/profile_handler.go
+#### Day1 cases only do creation step, we have to define a day1 case to verify the day1 works well
+* Create the case in terraform-provider-rhcs/tests/e2e/<feature name>_test.go
+* Label the case with ***CI.Day1Post***
+* Label the case with importance ***CI.Critical***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
+* Code for checking steps  only in the case
+### Contribute to day2
+* Create the case in terraform-provider-rhcs/tests/e2e/<feature name>_test.go
+* Label the case with ***CI.Day2***
+* Label the case with importance ***CI.Critical*** or ***CI.High***
+* Don't need to run creation step, just in BeforeEach step call function  ***PrepareRHCSClusterByProfileENV()*** it will load the clusterID prepared
```

Comment:
Should I add PrepareRHCSClusterByProfileENV() before each TC? I thought the cluster should be prepared as previous step, and Day 2 cases rely that the cluster exist.

Reply:
@amalykhi As the doc tell `PrepareRHCSClusterByProfileENV` will load the cluster prepared in pre step. It won't create cluster again. You can put it to beforeeach, or we can make it a global variable.

Reply:
@amalykhi in that case we will need to change our automated test cases later on.
but let's keep them like that for now (as it is done [here](https://github.com/terraform-redhat/terraform-provider-rhcs/blob/2fb06bc27ee85dcde09425bf519cb15b1dde2d44/tests/e2e/init_test.go#L19))
and once this PR merged we will change ours as well on a new PR.